ext/unf_ext/extconf.rb

```diff
@@ -15,6 +15,14 @@
     # Ruby < 2.0
     $CFLAGS << ' -fsigned-char'
   end
+when /\Aaarch64/
```

Comment:
Since the code is the same in `when /\Aarm/`and `when /\Aaarch64/`, they should be merged as a single `when /\A(arm|aarch64)/`.